### DISCUSSION TITLE
[HARDWARE] CPU support

### DISCUSSION THREAD

**Discussion by ray2022** [hlky/stable-diffusion-webui#212] [General]
It would be great to be able to run this on CPU only.

I was able to run https://github.com/loadletter/stable-diffusion on CPU and it works good but it's a simple version without webui.

I think CPU support was one of the changes in that pull request: https://github.com/hlky/stable-diffusion-webui/pull/57

**Comment by hlky** [comment]
Thanks for narrowing down the required changes, will make it easier to implement

**Comment by Khyta** [comment]
When is CPU support coming? I cannot afford an Nvidia GPU but I would really like to be able to use it on my Laptop.
